internal/test/mockRDSClient/mock_rds_client.go

```diff
@@ -1,4 +1,4 @@
-package cmd
+package mockrdsclient
```

Comment:
Just as a suggestion, we might prefer to have the `mocks` package under `internal/test` and this one be called `mock_rds_client.go`, with the type `mockRDSClient`. What do you think?